apps/website/src/components/layout/footer/Footer.tsx

```diff
@@ -12,22 +16,41 @@ export const Footer: React.FC = () => {
           <ul className="flex flex-wrap items-center justify-between">
             <li className="basis-full p-2 md:basis-1/3">
               <p>
-                Built by the Alveus.gg team and community,
-                <br />
+                Original design by{" "}
```

Comment:
```suggestion
                Original design by
                {" "}
```
Nit, so welcome to disagree, but I prefer putting these on a newline when I need them

Reply:
Yeah i do not like what prettier does here at all with the newlines, but I have it set to run on save. Not sure if I should disable it just for this ...

Reply:
I'll just put a /* prettier-ignore */ in front of the jsx block!

Reply:
(I'll be honest, TIL that prettier was being used here, I don't think I've been running it)